models/base_model.py

```diff
@@ -38,33 +38,11 @@ class BaseModel:
 
     def __init__(self, *args, **kwargs):
         """instantiation of new BaseModel Class"""
-
+        self.id = str(uuid4())
+        self.created_at = datetime.now()
         if kwargs:
-            self.__set_attributes(kwargs)
```

Comment:
we removed the __set_attributes method because it was an unnecessary method. the purpose of that method was to catch to catch the datetime string and convert it to an object, but we moved that to the reload() function in the filestorage